pyproject.toml

```diff
@@ -11,8 +11,8 @@ pandas = "^2.0.2"
 mypy = "^1.4.1"
 pydantic = "^1.10.9"
 pandas-stubs = "^2.0.2.230605"
```

Comment:
Two issues here, first is you don't want to remove networkx/distinctipy, second is `sklearn` is https://pypi.org/project/scikit-learn/ and should be installed as `scikit-learn`.

Reply:
ok should I readd those then do a another pull request?

Reply:
I took care of it Brenda! it was an easy fix